ColabTool/Shared/Post/story.jsx

```diff
@@ -152,7 +152,7 @@ storiesOf('Post')
     <Post
       hasPermission
       draftDetails={isARetweetDraftDetails}
-      links={links}
+      retweetCommentLinks={links}
```

Comment:
Heya @hharnisc 😄 

Added `retweetCommentLinks` since we'll need both link info for the comment text as well as the retweet text.

From buffer-web we'll be sending the retweet text as the `text` so the regular `links` prop will be for that.